shell/platform/darwin/ios/framework/Source/SemanticsObject.mm

```diff
@@ -9,6 +9,8 @@
 
 namespace {
 
+constexpr float kScrollExtentMaxForInf = 1000;
```

Comment:
It's probably worth commenting why 1000.  That sounds like a small number, it's hard for me to know if it's adequate.